packages/presentation/package.json

```diff
@@ -55,7 +55,8 @@
     "@hcengineering/collaborator-client": "^0.6.4",
     "fast-equals": "^5.2.2",
     "png-chunks-extract": "^1.0.0",
-    "uuid": "^8.3.2"
+    "uuid": "^8.3.2",
+    "@hcengineering/workbench": "^0.6.16"
```

Comment:
Nope!!!!

Reply:
why?

Reply:
The workbench is almost empty. I can't do the reverse due to the fact that presentation has svelte components.

Reply:
core -> presentation -> workbench, we could have any other starting apps, workbench is just one of them.

Reply:
core should depend only on core
packages should depend only on packages and core, if we have some of deps they need to be eliminated. 
And of cause we should not add more.

Reply:
there are literally no other place to put it then atm 😄
i can only suggest leaving the fn in text-editor-resources, which is kinda strange, but...

Reply:
the problem is that text-editor-resources is the actual core. Almost all packages that have Svelte components depend on it, so if something is needed there, it usually stays there, regardless of where it would be more appropriate to put it, because otherwise there will be a cyclic dependency

Reply:
though probably the idea was that we should avoid dependencies between *-resource packages, and the fact that there are any at all is something that needs to be addressed.